creator-node/src/fileManager.js

```diff
@@ -177,10 +177,14 @@ async function fetchFileFromNetworkAndWriteToDisk({
   // this is the replacement for the old findCIDInNetwork call
   // short circuit calling the rest of the network if env var is not enabled or
   // the rest of the network has already been checked the network today
-  if (!config.get('findCIDInNetworkEnabled')) return false
+  if (!config.get('findCIDInNetworkEnabled'))
+    throw new Error(`findCIDInNetworkEnabled = false`)
 
   const attemptedStateFix = await getIfAttemptedStateFix(storageLocation)
-  if (attemptedStateFix) return
+  if (attemptedStateFix)
+    throw new Error(
+      `Already attempted to find from network; Currently cannot try again`
+    )
```

Comment:
BUGFIX: both of these conditions need to throw an error out to the outer function isntead of silently returning, since currently the outer function assumes success and server tries to stream the file at this path even tho it doesn't exist

i confirmed that by throwing this error, things are handled correctly

Reply:
solid catch